### PR TITLE
Dala 3178 introduce premium user role

### PR DESCRIPTION
**Pull request by BenediktLaubmann**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] The version of main currently active on prod is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] The data from prod is migrated via DB dumps (see Wiki page on "Server Migration" for details) 
  - [ ] The feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
- [ ] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -51,9 +51,9 @@ jobs:
           cache: gradle
       - name: Frontend component testing
         run: timeout -v 15m ./testing/execute_frontend_component_tests.sh
-# Uncomment the lines below to enable debug logging for investigating flaky component tests.
-#        env:
-#          DEBUG: cypress:*
+        # Uncomment the lines below to enable debug logging for investigating flaky component tests.
+        #        env:
+        #          DEBUG: cypress:*
```
Comment (PaulChristiansen) [inline .github/workflows/CI.yaml:56]: Additional indentation can probably be reverted

File: .github/workflows/CI.yaml [hunk 2]
```diff
@@ -187,7 +187,7 @@ jobs:
           vars: ${{ toJSON(vars) }}
       - name: Prepare python environment
         if: always()
-        run: | 
+        run: |
```
Comment (PaulChristiansen) [inline .github/workflows/CI.yaml:190]: Also probably not necessary

File: .github/workflows/CI.yaml (hunk 1 above)
Comment (BenediktLaubmann) [reply .github/workflows/CI.yaml:56]: ah yes, this was my auto-linter

File: .github/workflows/CI.yaml (hunk 2 above)
Comment (BenediktLaubmann) [reply .github/workflows/CI.yaml]: also reverted

File: .github/workflows/CI.yaml (hunk 1 above)
Comment (BenediktLaubmann) [reply .github/workflows/CI.yaml]: reverted

**Pull request merged**